### ISSUE TITLE
Installation from repo head

### ISSUE THREAD

**Issue by Jackymancs4** [makeworld-the-better-one/homebrew-tap#5]
May I ask you the reasons for 54ceb1290408b78b89f56b9812496f0b6cdfb81c ?

**Comment by Jackymancs4** [comment]
For me it's very important, because for example an issue of mine was fixed here https://github.com/makeworld-the-better-one/amfora/commit/5cb629f4c32c79b5544c572caf5796487bf03cf8 at the beginning of August but the next release came out almost a month after that. In the meantime I had to use the head build of my formula to be able to properly use the tabs feat.

**Comment by makew0rld** [comment]
It definitely is important. I removed it because I couldn't get the makefile to work correctly. The commit and version info in the binary would be completely blank:
```
$ amfora -v
Amfora
Commit:
Built by: official-brew-tap
```

I believe this is related to how homebrew clones git repos, but I'm not sure. Maybe it doesn't make the `.git` folder available to the script? I'd be happy to have solutions for this.

**Comment by Jackymancs4** [comment]
I have managed to create a formula that behave the way you want: having `amfora -v` to return
```
Amfora v1.5.0
Commit: 922e7981a92cb7bf0d7b3baf1694d0fffe90d448
Built by: official-brew-tap
```
when built normally and

```
Amfora v1.5.0-29-gfe0fd78
Commit: fe0fd784bb47ffe885c64baa91f18e50f3a54b96
Built by: official-brew-tap
```

when built from head. You can find the final formula here:
https://gist.github.com/Jackymancs4/83730fd42b96c9715ce1a09958a6d5f4

I honestly fail to get why make works the way it does, but I've very little experience about it.

**Comment by makew0rld** [comment]
Fixed by #6.
